i18n/ro.json

```
@@ -3024,7 +3024,7 @@
   "help.formatting.intro": "Markdown facilitează formatarea mesajelor. Tastați un mesaj așa cum ați proceda în mod normal și utilizați aceste reguli pentru al face cu formatare specială.",
   "help.formatting.lines.description": "Creează o linie utilizând trei `*`, `_` sau `-`.",
   "help.formatting.lines.title": "Linii",
-  "help.formatting.linkEx": "[Check out Mattermost!] (Https://about.mattermost.com/)",
+  "help.formatting.linkEx": "<link>Check out Mattermost!</link> (Https://about.mattermost.com/)",
```

Also fixing a bug here since the Markdown was broken in the previous version because of the space in it
```suggestion
  "help.formatting.linkEx": "[Check out Mattermost!](https://about.mattermost.com/)",
```